pyscript.core/src/exceptions.js

```diff
@@ -33,19 +43,35 @@ export class UserError extends Error {
 }
 
 export class FetchError extends UserError {
+    /**
+     * @param {ErrorCodes} errorCode
+     * @param {string} message
+     * */
     constructor(errorCode, message) {
         super(errorCode, message);
         this.name = "FetchError";
     }
 }
 
 export class InstallError extends UserError {
+    /**
+     * @param {ErrorCodes} errorCode
+     * @param {string} message
+     * */
     constructor(errorCode, message) {
         super(errorCode, message);
         this.name = "InstallError";
     }
 }
 
+/**
+ * Internal function for creating alert banners on the page
+ * @param {string} message
+ * @param {string} level
+ * @param {string} messageType="text"
```

Comment:
wheren't optional args supposed to be in `[squared_brackets]` ??? just curiosity

Reply:
I found some conflicting documentation on this, but square brackets are apparently correct for tsc's interpretation of jsdoc. Thank you!